common/measurement.cpp

```diff
@@ -334,7 +335,7 @@ void measurement::update_input(ux_window& win, const rs2::rect& viewer_rect)
 {
     id = 0;
 
-    if (ImGui::IsKeyPressed('Z') || ImGui::IsKeyPressed('z'))
+    if (ImGui::IsKeyPressed(ImGuiKey_Z))
```

Comment:
Win this handle uppercase & lowercase letters?

Reply:
Yes, this only checks if the Z key is pressed (will work uppercase and lowercase letters).